assets/js/base/components/cart-checkout/totals/totals-taxes-item/index.js

```diff
@@ -3,26 +3,44 @@
  */
 import { __ } from '@wordpress/i18n';
 import PropTypes from 'prop-types';
-import { TAXES_ENABLED } from '@woocommerce/block-settings';
+import {
+	TAXES_ENABLED,
+	DISPLAY_ITEMIZED_TAXES,
+} from '@woocommerce/block-settings';
 
 /**
  * Internal dependencies
  */
 import TotalsItem from '../totals-item';
 
 const TotalsTaxesItem = ( { currency, values } ) => {
-	const { total_tax: totalTax } = values;
+	const { total_tax: totalTax, tax_lines: taxLines } = values;
 
 	if ( ! TAXES_ENABLED ) {
 		return null;
 	}
 
+	if ( ! DISPLAY_ITEMIZED_TAXES ) {
```

Comment:
Do we need this option? I think it we can always return `tax_lines` as an array (even if it has one) since this logic already lives in the backend

Reply:
On a second look, it doesn’t look like it’s included, separate lines will always be returned, we can probably change `get_tax_lines` to return a total in that case or return all items, and skip `total_tax` in the API, but I'm not sure  I like this as well.

As a note, this is just a suggestion, not a blocker.

Reply:
I prefer using the totals field as that is what the field is intended for, and to note, it might have a different "label" than the tax lines field.